deps/glu.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "glu",
```

Comment:
`glu` is in https://github.com/flathub/shared-modules

Reply:
ok